test/integration_test/migration_dr_actions_test.go

```diff
@@ -0,0 +1,548 @@
+	sourceStatefulsetReplicas := *sourceStatefulsets.Items[0].Spec.Replicas
+
+	// Create the clusterpair
+	clusterPairNamespace := defaultAdminNamespace
+	if !unidirectionalClusterpair {
+		log.Info("Bidirectional flag is set, will create bidirectional cluster pair:")
+		log.InfoD("Name: %s", remotePairName)
+		log.InfoD("Namespace: %s", clusterPairNamespace)
+		log.InfoD("Backuplocation: %s", defaultBackupLocation)
+		log.InfoD("Secret name: %s", defaultSecretName)
+		err = scheduleBidirectionalClusterPair(remotePairName, clusterPairNamespace, projectIDMappings, defaultBackupLocation, defaultSecretName)
+		log.FailOnError(t, err, "failed to set bidirectional cluster pair: %v", err)
+		err = setSourceKubeConfig()
+		log.FailOnError(t, err, "failed to set kubeconfig to source cluster: %v", err)
+	} else {
+		log.Info("Creating Unidirectional flag is set, will create unidirectional cluster pair:")
+		log.InfoD("Name: %s", remotePairName)
+		log.InfoD("Namespace: %s", clusterPairNamespace)
+		log.InfoD("Backuplocation: %s", defaultBackupLocation)
+		log.InfoD("Secret name: %s", defaultSecretName)
+		err = scheduleUnidirectionalClusterPair(remotePairName, clusterPairNamespace, projectIDMappings, defaultBackupLocation, defaultSecretName, true, false)
+		log.FailOnError(t, err, "failed to set unidirectional cluster pair: %v", err)
+	}
+
+	// Create the migration schedule
+	// we want to create the schedulePolicy and migrationSchedule using storkctl instead of scheduling apps using torpedo's scheduler
+	// schedulePolicyArgs is a map of schedulePolicyName : {{flag1:value1,flag2:value2,....}}
+	schedulePolicyArgs := make(map[string]map[string]string)
+	schedulePolicyArgs["migrate-every-5m"] = map[string]string{"policy-type": "Interval", "interval-minutes": "5"}
+
+	// migrationScheduleArgs is a map of migrationScheduleName : {{flag1:value1,flag2:value2,....}}
+	migrationScheduleArgs := make(map[string]map[string]string)
+	migrationScheduleArgs[instanceID] = map[string]string{
+		"purge-deleted-resources": "",
+		"schedule-policy-name":    "migrate-every-5m",
+	}
+
+	//Create schedulePolicies using storkCtl if any required
+	factory := storkctl.NewFactory()
+	var outputBuffer bytes.Buffer
+	cmd := storkctl.NewCommand(factory, os.Stdin, &outputBuffer, os.Stderr)
+	for schedulePolicyName, customArgs := range schedulePolicyArgs {
+		cmdArgs := []string{"create", "schedulepolicy", schedulePolicyName}
+		executeStorkCtlCommand(t, cmd, cmdArgs, customArgs)
+	}
+	//Create migrationSchedules using storkCtl
+	migrationScheduleName := "forward-migration-schedule-multiple-ns"
+	namespacesValue := fmt.Sprintf("%s,%s", mysqlNamespace, elasticsearchNamespace)
+	cmdArgs := []string{"create", "migrationschedule", migrationScheduleName, "-c", remotePairName,
+		"--namespaces", namespacesValue, "-n", defaultAdminNamespace}
+	executeStorkCtlCommand(t, cmd, cmdArgs, migrationScheduleArgs[migrationKey])
```

Comment:
shouldn't this be migrationScheduleArgs[instanceID] instead of migrationScheduleArgs[migrationKey] ??

Reply:
Done